samples/notebooks/prediction/SDK_Custom_Predict_SDK_Integration.ipynb

```diff
@@ -0,0 +1,1311 @@
+    "\n",
+    "try:\n",
+    "    PROJECT_ID = shell_output[0]\n",
+    "except IndexError:\n",
+    "    PROJECT_ID = None\n",
+    "\n",
+    "print(\"Project ID:\", PROJECT_ID)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "qJYoRfYng0XZ"
+   },
+   "source": [
+    "Otherwise, set your project ID here."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "riG_qUokg0XZ"
+   },
+   "outputs": [],
+   "source": [
+    "if PROJECT_ID == \"\" or PROJECT_ID is None:\n",
+    "    PROJECT_ID = \"[PROJ_ID]\"  # @param {type:\"string\"}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "XoEqT2Y4DJmf"
+   },
+   "source": [
+    "### Configure project and resource names"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "MzGDU7TWdts_"
+   },
+   "outputs": [],
+   "source": [
+    "REGION = \"us-central1\"  # @param {type:\"string\"}\n",
+    "MODEL_ARTIFACT_DIR = \"sklearn-cpr-model\"  # @param {type:\"string\"}\n",
+    "REPOSITORY = \"custom-container-prediction\"  # @param {type:\"string\"}\n",
+    "IMAGE = \"sklearn-cpr-server\"  # @param {type:\"string\"}\n",
+    "MODEL_DISPLAY_NAME = \"sklearn-cpr-model\"  # @param {type:\"string\"}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ca1a915d641d"
+   },
+   "source": [
+    "`REGION` - Used for operations\n",
+    "throughout the rest of this notebook. Make sure to [choose a region where Cloud\n",
+    "Vertex AI services are\n",
+    "available](https://cloud.google.com/vertex-ai/docs/general/locations#feature-availability). You may\n",
+    "not use a Multi-Regional Storage bucket for prediction with Vertex AI.\n",
+    "\n",
+    "`MODEL_ARTIFACT_DIR` - Folder directory path to your model artifacts within a Cloud Storage bucket, for example: \"my-models/fraud-detection/trial-4\"\n",
+    "\n",
+    "`REPOSITORY` - Name of the Artifact Repository to create or use.\n",
+    "\n",
+    "`IMAGE` - Name of the container image that will be pushed.\n",
+    "\n",
+    "`MODEL_DISPLAY_NAME` - Display name of Vertex AI Model resource."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "62f861b68b50"
+   },
+   "source": [
+    "### Create a Cloud Storage bucket\n",
+    "\n",
+    "**The following steps are required, regardless of your notebook environment.**\n",
+    "\n",
+    "To update your model artifacts without re-building the container, you must upload your model\n",
+    "artifacts and any custom code to Cloud Storage.\n",
+    "\n",
+    "Set the name of your Cloud Storage bucket below. It must be unique across all\n",
+    "Cloud Storage buckets and start with `gs://`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "9724b00aeead"
+   },
+   "outputs": [],
+   "source": [
+    "BUCKET_NAME = \"[BUCKET]\"  # @param {type:\"string\"}"
```

Comment:
Let's rename to gs://[BUCKET]. Here and the other M4 notebook.

I created a PR to change for the M2 notebook already (https://github.com/googleapis/python-aiplatform/pull/1045).

Reply:
Done.